app/views/directives/traffic-table.html

```diff
@@ -14,9 +14,9 @@
   <tbody ng-if="(portsByRoute | hashSize) == 0">
     <tr><td colspan="7"><em>{{emptyMessage || 'No routes or ports to show'}}</em></td></tr>
   </tbody>
-  <tbody ng-if="(portsByRoute | hashSize) > 0">
-    <tr ng-repeat-start="(routeName,ports) in portsByRoute" style="display:none;"></tr>
-    <tr ng-repeat="port in ports" ng-if="routeName !== ''">
+  <tbody ng-if="(portsByRoute | hashSize) > 0" ng-repeat-start="(routeName,ports) in portsByRoute" style="display:none;"></tbody>
+  <tbody ng-repeat="port in ports" ng-if="routeName !== ''">
```

Comment:
I think we have the 2px border on other responsive tables. Can you check? If so, let's open a separate issue to fix them all.

cc @rhamilto @sg00dwin